apps/modernization-ui/src/apps/page-builder/components/CreateQuestion/CreateNumericQuestion.tsx

```diff
@@ -0,0 +1,99 @@
+    CreateDateQuestionRequest &
+    CreateTextQuestionRequest &
+    ReportingInfo &
+    MessagingInfo &
+    QuestionFormType;
+
+type NumericQuestionProps = {
+    control?: Control<CreateQuestionFormType, any>;
+};
+
+export const CreateNumericQuestion = ({ control }: NumericQuestionProps) => {
+    return (
+        <>
+            <Controller
+                control={control}
+                name="minValue"
+                rules={{
+                    required: { value: true, message: 'Minimum Value required' },
+                    maxLength: 50
+                }}
+                render={({ field: { onChange, value }, fieldState: { error } }) => (
+                    <Input
+                        onChange={onChange}
+                        className="field-space"
+                        value={value}
+                        label="Minimum Value"
+                        type="number"
+                        error={error?.message}
+                        required
+                    />
+                )}
+            />
+            <Controller
+                control={control}
+                name="maxValue"
+                rules={{
+                    required: { value: true, message: 'Maximum Value required' },
+                    maxLength: 50
+                }}
+                render={({ field: { onChange, value }, fieldState: { error } }) => (
+                    <Input
+                        onChange={onChange}
+                        className="field-space"
+                        value={value}
+                        label="Maximum Value"
+                        type="number"
+                        error={error?.message}
+                        required
+                    />
+                )}
+            />
+            <label>
+                Related units <span className="mandatory-indicator">*</span>
+            </label>
+            <Controller
+                control={control}
+                name="relatedUnits"
+                defaultValue="No"
+                render={({ field: { onChange, value } }) => (
+                    <div className="radio-group">
+                        <Radio
+                            id="relatedUnits_Y"
+                            name="relatedUnits"
+                            value="Yes"
+                            label="Yes"
+                            onChange={(e: any) => onChange(e.target.value)}
+                            checked={value === 'Yes'}
+                        />
+                        <Radio
+                            id="relatedUnits_N"
+                            name="relatedUnits"
+                            value="No"
+                            label="No"
+                            onChange={(e: any) => onChange(e.target.value)}
+                            checked={value === 'No'}
+                        />
+                    </div>
+                )}
+            />
+        </>
```

Comment:
Issues (blocking): 
1. You cannot enter data into the `Field length`, `Minimum Value`, `Maximum Value` fields.
2. The `mask` options are not present (the valid mask values for a text question are available on the `mask` enum in the `CreateNumericQuestionRequest.ts`). The text to display for the options can come from Classic 
![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/08578ba6-f142-4c34-ae26-f016e26f9ab0)



3. The `Related units` radio button should display a `Unit type` selection when `Yes` is selected. 
![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/35be2f75-0d24-427d-82bb-ca403b75a202)